rdkit/Chem/RegistrationHash.py

```diff
@@ -127,13 +126,20 @@ def GetMolLayers(original_molecule: Chem.rdchem.Mol, data_field_names: Optional[
   mol = _RemoveUnnecessaryHs(original_molecule, preserve_stereogenic_hs=True)
   _StripAtomMapLabels(mol)
 
+  cxflag = Chem.CXSmilesFields.CX_ATOM_LABELS | Chem.CXSmilesFields.CX_ENHANCEDSTEREO
+  ps = Chem.SmilesWriteParams()
+
   Chem.CanonicalizeEnhancedStereo(mol)
 
   formula = rdMolHash.MolHash(mol, rdMolHash.HashFunction.MolFormula)
-  cxsmiles = Chem.MolToCXSmiles(mol)
+
+  cxsmiles = Chem.MolToCXSmiles(
+    mol, ps, cxflag)
+
   tautomer_hash = GetStereoTautomerHash(mol)
 
-  canonical_smiles = GetCanonicalSmiles(cxsmiles)
+  canonical_smiles = cxsmiles
```

Comment:
`canonical_smiles` is redundant, we can just use `cxsmiles`

Reply:
Yep, good point! Will do 👍

Reply:
Done!